internal/helpers/index.js

```diff
@@ -7,6 +7,9 @@ import * as path from 'node:path';
 export const itwinuiReactAliases =
   process.env.NODE_ENV === 'development'
     ? {
+        '@itwin/itwinui-react/styles.css?url': path.resolve(
+          '../../packages/itwinui-react/styles.css?url',
+        ),
         '@itwin/itwinui-react/styles.css': path.resolve(
           '../../packages/itwinui-react/styles.css',
```

Comment:
Since these two are very similar, maybe we could do a partial find/replace instead? https://www.npmjs.com/package/@rollup/plugin-alias#regular-expression-aliases

Reply:
I just realized a similar problem would happen when importing from `@itwin/itwinui-react/react-table`, so it's probably better do a partial replace, which will ensure everything works.

Reply:
Sounds good, I switched to using the find and replacement (https://github.com/iTwin/iTwinUI/pull/2038/commits/853223eb78f045e167b9baa5b612ecbf2aa61943).

I tested that it works for react-table too by adding `import type { Column } from '@itwin/itwinui-react/react-table';` to `App.tsx` from the vite playground. This worked without a separate rule for react-table ([rule](https://github.com/iTwin/iTwinUI/pull/2038/commits/0fb67ef69e4e39a9197680ff4029ebb3b6732578#diff-f4d8f020910bbda54d6058e9d99cbdeba33ded5431bc2807fb6bcf45630f66c6L22-L27)). However, doing something like `import '@itwin/itwinui-react/react-table';` sometimes gives errors:

<details>
<summary>Error</summary>

<img width="905" alt="image" src="https://github.com/iTwin/iTwinUI/assets/45748283/5719a25f-8a98-4f16-90a3-eeee4404f036">

</details>

So, I add a specific rule for `react-table` instead of a general catch-all rule (https://github.com/iTwin/iTwinUI/pull/2038/files/0fb67ef69e4e39a9197680ff4029ebb3b6732578..19e46b5901c4506ce1934d5758a589a2be99b532).

Reply:
To be clear, I'm suggesting something like this:

```js
{
  find: /^@itwin\/itwinui-react\/(.*)/,
  replacement: path.resolve(
    '../../packages/itwinui-react/$1',
  ),
},
{
  '@itwin/itwinui-react': path.resolve(
    '../../packages/itwinui-react/src',
  ),
},
```

Reply:
Ohh, I see, something like that works in the docs website but sometimes gives an error when doing `import '@itwin/itwinui-react/react-table';`. But the real case of importing types (e.g. `import type { Column } from '@itwin/itwinui-react/react-table';`) works.

So, I made changes similar to your suggestion (https://github.com/iTwin/iTwinUI/pull/2038/commits/3ff1f9581c8bd24383f17bc5f3a8c3781e073949).